Containerfile.openshift

```diff
@@ -0,0 +1,3 @@
+FROM overridden
```

Comment:
Can you add a comment here about what's going on?  I assume it's being configured in a Prow job?

Reply:
Added more details in the `Containerfile`!

Prow currently does not support multi-arch. The end goal is to have it built by ART (via OSBS/eventually Konflux) like any other image, but for now we'll just build it ourselves. That said, I've added a better default `FROM` image (which doesn't exist yet, but hopefully will soon).